spyder/utils/tests/test_programs.py

```diff
@@ -37,37 +43,15 @@ def test_run_python_script_in_terminal_with_wdir_empty(tmpdir, qtbot):
     res = outfilepath.read()
     assert res == 'done'
 
-@pytest.mark.skipif(os.environ.get('TEST_CI_WIDGETS', None) == True,
-                    reason='To only run the test in spyders CI services.')
-def test_is_python_interpreter_linux_true():
-    filename = "$HOME/miniconda/bin/python"
-    valid = is_python_interpreter(filename)
-
-    assert valid == True
-
-@pytest.mark.skipif(os.environ.get('TEST_CI_WIDGETS', None) == True,
-                    reason='To only run the test in spyders CI services.')
-def test_is_python_interpreter_linux_false():
-    filename = "$HOME/miniconda/bin/ipython"
-    valid = is_python_interpreter(filename)
-
-    assert valid == False
-
-@pytest.mark.skipif(os.environ.get('TEST_CI_WIDGETS', None) == True,
-                    reason='To only run the test in spyders CI services.')
-def test_is_python_interpreter_windows_true():
-    filename = "%PYTHON%\bin\python.exe"
-    valid = is_python_interpreter(filename)
-
-    assert valid == True
-
-@pytest.mark.skipif(os.environ.get('TEST_CI_WIDGETS', None) == True,
-                    reason='To only run the test in spyders CI services.')
-def test_is_python_interpreter_windows_false():
-    filename = "%PYTHON%\Scripts\ipython.exe"
-    valid = is_python_interpreter(filename)
+@pytest.mark.skipif(os.environ.get('CI', None) == True,
```

Comment:
The problem is these tests are running locally (which is what I wanted to prevent)

@mariacamilaremolinagutierrez, please fix that.